tests/fixtures/config.ts

```diff
@@ -0,0 +1,11 @@
+export const environment = [
+  {
+    CURRENT_ENVIRONMENT: "production",
+  },
+  {
+    CURRENT_ENVIRONMENT: "staging",
+  },
+  {
+    CURRENT_ENVIRONMENT: "",
```

Comment:
can we use `"default"` string here ?

Reply:
We can but I wanted to test specifically for empty key therefore I have added the fixtures this way.